src/services/index.ts

```diff
@@ -1,12 +1,13 @@
-import { BoardInfo } from '../types';
-import { mockService } from './mock';
+import { BoardInfo, BoardID, CreateBoardBody } from '../types';
+import { remoteService } from './remote';
 
 export const service = getAPIMethod();
 
 function getAPIMethod() {
-  return mockService();
+  return remoteService();
 }
 
 export interface Service {
   getBoardDetail(boardID: string): Promise<Pick<BoardInfo, 'title' | 'savedTime'>>;
+  createBoard(body: CreateBoardBody): Promise<BoardID>;
```

Comment:
BoardID 타입은 없어도 될 것 같습니다!
그리고 성공 여부를 담아주고 그걸로 실패 시 화면 이동이 안되는 등 처리를 해주면 좋을 것 같아요!
```suggestion
  createBoard(body: CreateBoardBody): Promise<{ boardID: string, isSuccess: boolean }>;
```